### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by devypt**

### CODE REVIEW

File: controllers/ProjectController.php [hunk 1]
```diff
@@ -13,81 +13,89 @@
...
+    /**
+     * @return array
+     */
     public function behaviors()
     {
         return [
             'access' => [
                 'class' => AccessControl::className(),
-                'only' => ['create', 'update', 'delete-image'], //only be applied to
+                'only'  => ['create', 'update', 'delete-image'], //only be applied to
```
Comment (samdark) [inline controllers/ProjectController.php]: Why extra spaces here and in other cases?

File: controllers/ProjectController.php [hunk 2]
```diff
@@ -13,81 +13,89 @@
...
     }
 
+    /**
+     * @return string
+     */
     public function actionIndex()
     {
+        $limit = Yii::$app->params['project.pagesize'];
+
+        $project          = new Project();
```
Comment (samdark) [inline controllers/ProjectController.php]: It is not correct. Model instance should not manage queries, it represents single a row in the database.

Comment (samdark) [inline controllers/ProjectController.php]: Refactoring itself makes sense but I'd move method to ProjectQuery static method instead.

File: controllers/ProjectController.php (hunk 1 above)
Comment (devypt) [inline controllers/ProjectController.php]: @samdark i knew it's hard to review the code with extra space and formatting but if you checked it in your IDE you will find code format now it's better, the main change is array elements alignment

Comment (devypt) [inline controllers/ProjectController.php]: @samdark i knew it's hard to review the code with extra space and formatting but if you checked it in your IDE you will find code format now it's better, the main change is array elements alignment

File: controllers/ProjectController.php (hunk 2 above)
Comment (devypt) [inline controllers/ProjectController.php]: ok, i will move it to ProjectQuery

File: controllers/ProjectController.php (hunk 1 above)
Comment (samdark) [inline controllers/ProjectController.php]: No, it's not better :)

File: models/Image.php [hunk 3]
```diff
@@ -166,27 +202,33 @@ public function ensureThumbnailPath()
     public function generateThumbnail($crop = null)
     {
         ImagineImage::$thumbnailBackgroundAlpha = 0;
-        $size = Yii::$app->params['image.size.thumbnail'];
-        
+        $size                                   = Yii::$app->params['image.size.thumbnail'];
```
Comment (lex111) [inline models/Image.php]: Just my two cents:

Such an alignment, like here, does not look very pretty, do not you think? I think we need to compile a file describing the code style used, which should be respected by all participants. For example, I was more inclined to use "old" formatting, it is more familiar.

**Pull request closed without merging**